client/my-sites/plans/jetpack-plans/product-card/index.tsx

```diff
@@ -240,6 +244,11 @@ const ProductCard: React.FC< ProductCardProps > = ( {
 			scrollCardIntoView={ scrollCardIntoView }
 			collapseFeaturesOnMobile={ collapseFeaturesOnMobile }
 			pricesAreFetching={ pricesAreFetching }
+			isPricingPageTreatment202204={ isPricingPageTreatment202204 }
+			isPricingPageTest202204AssignmentLoading={ isPricingPageTest202204AssignmentLoading }
+			belowButtonText={
+				isPricingPageTreatment202204 ? translate( 'Renews at the normal rate.' ).toString() : ''
```

Comment:
I'm late to the party, but translate has a second parameter where you can add an options object, indicating that you want the result to be a string.

```suggestion
				isPricingPageTreatment202204 ? translate( 'Renews at the normal rate.', { textOnly: true } ): ''
```

Forgive any mistake I might have, I'm on my mobile phone :)